plugins/synaptics-mst/fu-synaptics-mst-firmware.c

```diff
@@ -59,14 +114,33 @@ fu_synaptics_mst_firmware_parse(FuFirmware *firmware,
 static GByteArray *
 fu_synaptics_mst_firmware_write(FuFirmware *firmware, GError **error)
 {
+	FuSynapticsMstFirmware *self = FU_SYNAPTICS_MST_FIRMWARE(firmware);
 	g_autoptr(GByteArray) buf = g_byte_array_new();
 	g_autoptr(GBytes) blob = NULL;
-
+	guint16 addr;
+
+	switch (self->family) {
+	case FU_SYNAPTICS_MST_FAMILY_TESLA:
+	case FU_SYNAPTICS_MST_FAMILY_LEAF:
+	case FU_SYNAPTICS_MST_FAMILY_PANAMERA:
+		addr = ADDR_CUSTOMER_ID;
```

Comment:
Maybe addr_customer_id_tesla to follow the naming convention used for Cayenne.

Reply:
Done